classes/02class/exercises/c02-network01/afaryy/README.md

```diff
@@ -0,0 +1,48 @@
+# c02-network01
+
+## Commands Execution Output
+
+- Answers for `10.0.0.1/24`:
+
+  - IP in binary: `00001010.00000000.00000000 .00000001`
+  - Netmask in binary: `11111111.11111111.11111111.00000000` 
+  - Netmask in decimals: `255.255.255.0`
+  - private/public: `private`
+  - hosts: `254` (10.0.0.1-10.0.0.254)
+
+- Answers for `192.168.0.1/16`:
+
+  - IP in binary: `11000000.10101000 .00000000.00000001`
+  - Netmask in binary: `11111111.11111111 .00000000.00000000` 
+  - Netmask in decimals: `255.255.0.0`
+  - private/public: `private`
+  - hosts: `65534 ` (192.168.0.1/16-192.168.255.254)
+
+- Answers for `249.165.166.135/30`:
+
+  - IP in binary: `11111001.10100101.10100110.100001 11`
+  - Netmask in binary: `11111111.11111111.11111111.111111 00` 
+  - Netmask in decimals: `255.255.255.252`
+  - private/public: `public`
+  - hosts: `2` (249.165.166.133-249.165.166.134)
+
+- Answers for `236.68.223.18/32`:
+
+  - IP in binary: `11111001.10100101.10100110.100001 11`
+  - Netmask in binary: `11111111.11111111.11111111.11111111` 
+  - Netmask in decimals: `255.255.255.255`
+  - private/public: `public`
+  - hosts: `1` (236.68.223.19)
```

Comment:
`/32` is kind-of a special case. It does not declare a subnet but a single-host instead.

This is not a network and the only host would be `236.68.223.18` itself.

When we use something different than `/32` we mean "all IPs in that network (or subnet for short)".

When we use `/32` we mean "only that specific IP address"

Reply:
Thank you for your explaination.